<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountingWidget</class>
 <widget class="QWidget" name="AccountingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainLayout" stretch="0,10,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QGridLayout" name="buttonsLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="writeOffProductButton">
         <property name="text">
          <string>Списать товар</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QComboBox" name="columnsBox"/>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="sortButton">
         <property name="text">
          <string>Сортировать</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="moveProductButton">
         <property name="text">
          <string>Перенести товар</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="findEdit"/>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="filterButton">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="tableLayout" columnstretch="2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="horizontalSpacing">
        <number>7</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableView">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>140</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTextBrowser" name="informationBrowser"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="text">
            <string>Обновить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Сбросить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
